Binary files libretro-blastem.orig/.git/index and libretro-blastem/.git/index differ
diff -rupN libretro-blastem.orig/libretro/Makefile libretro-blastem/libretro/Makefile
--- libretro-blastem.orig/libretro/Makefile	2018-12-07 10:09:11.171250785 +0100
+++ libretro-blastem/libretro/Makefile	2018-12-07 10:19:39.402863430 +0100
@@ -1,7 +1,7 @@
 DEBUG := 0
 NAME  := blastem_libretro
 SOEXT := .so
-FLTO  :=
+FLTO  := -flto
 WITH_Z80      := 1
 WITH_DYNAREC  :=
 OPTIMIZE_FLAG :=
@@ -29,8 +29,8 @@ ifneq ($(SANITIZER),)
    LDFLAGS := -fsanitize=$(SANITIZER) $(LDLAGS)
 endif
 
-CFLAGS  := $(OPTIMIZE_FLAG) -std=gnu99 -Wall -D__LIBRETRO__ -I. -I.. -Ilibco $(SANITIZER)
-LDFLAGS := $(SANITIZER) -shared -Wl,--no-undefined
+CFLAGS  := $(CFLAGS) -std=gnu99 -Wall -D__LIBRETRO__ -I. -I.. -Ilibco
+LDFLAGS := $(LDFLAGS) -shared -Wl,--no-undefined
 LIBS    := -lm
 
 ifeq ($(platform),)
